sql/src/test/java/io/crate/integrationtests/RepositoryIntegrationTest.java

```diff
@@ -65,39 +63,17 @@ public static void deleteTempFolderRoot() throws Exception {
         TEMP_FOLDER_ROOT.delete();
     }
 
-    @Before
-    public void prepare() throws Exception {
-        client().admin().cluster()
-                .preparePutRepository(REPO_NAME)
-                .setType(FsRepository.TYPE)
-                .setSettings(ImmutableSettings.builder()
-                        .put("location", new File(temporaryFolder.getRoot(), REPO_NAME)))
-                .execute().actionGet(TimeValue.timeValueSeconds(2));
-        waitNoPendingTasksOnAll();
-    }
-
-    @After
-    public void cleanUp() throws Exception {
-        try {
-            client().admin().cluster().prepareDeleteRepository(REPO_NAME)
-                    .execute().actionGet(TimeValue.timeValueSeconds(2));
-        } catch (RepositoryMissingException e) {
-            // ignore
-        }
-    }
-
     @Test
     public void testDropExistingRepository() throws Exception {
-        execute("DROP REPOSITORY " + REPO_NAME);
+        execute("CREATE REPOSITORY existing_repo TYPE \"fs\" with (location=?, compress=True)",
+                new Object[]{TEMP_FOLDER_ROOT.getAbsolutePath()});
+        waitNoPendingTasksOnAll();
+        execute("DROP REPOSITORY existing_repo");
         assertThat(response.rowCount(), is(1L));
 
         waitNoPendingTasksOnAll();
-
-        expectedException.expect(RepositoryMissingException.class);
-        expectedException.expectMessage("[existing_repo] missing");
-        client().admin().cluster()
-                .prepareGetRepositories(REPO_NAME)
-                .execute().actionGet(TimeValue.timeValueSeconds(2));
+        execute("select * from sys.repositories where name = ? ", new Object[]{"existing_repo"});
+        assertThat(response.rowCount(), is(0L));
```

Comment:
shouldn't we still assert on the ES Api or at least query the cluster metadata to ensure its really dropped?